src/components/navigation/z-user-dropdown/styles.css

```
@@ -63,6 +64,7 @@ ul.inverse {
 ul > li {
   display: flex;
   justify-content: flex-end;
+  white-space: nowrap;
   padding: calc(var(--space-unit) * 1.5 - 1px) 0;
   margin: 0 calc(var(--space-unit) * 4) 0 0;
```

scrivo tutta la spiegazione qui: in mobile il box dell'`ul` non rispetta i padding del container, andando fuori dal box. la cosa si risolve mettendo `box-sizing: border-box;` all'`ul`. come conseguenza, il margin che era settato qui non serve più, perché i `li` finiscono già dove devono finire

```suggestion
  margin: 0;
```